### DISCUSSION TITLE
Dell 7472 instalar o SONOMA??

### DISCUSSION THREAD

**Discussion by Xibotrix** [community/community#135985] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Nao consigo instalar o sonoma no Meu Dell 7472, se algeuem tiver uma EFI que funciona no sonoma, pode me ajudar?

**Comment by LiteBrite82** [comment]
Hi @Xibotrix , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
